config.toml

```diff
@@ -8,6 +8,6 @@ highlight_code = true
 highlight_theme = "base16-ocean-light"
 
 [extra]
-logo = "https://easydocs.codeandmedia.com/logo.svg"
+logo = "https://github.com/forgeflux-org/docs/blob/main/static/logo.svg"
```

Comment:
please change this URL to `/logo.svg`, all files in `/static` dir are available at root

Reply:
Ah, very well. That will be done.